### PR TITLE
add page with related projects

### PR DESCRIPTION
**Pull request by jenshnielsen**
First version of a docs page with related projects. Please have a look and see if I missed anything

### CODE REVIEW

File: docs/community/related_projects.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+Broadbean can be installed with pip from PyPi
+with ``pip install qcodes_contrib_drivers``
+The source code is hosted on Github
+`here <https://github.com/qcodes/qcodes_contrib_drivers>`__
+and the documentation can be found
+`here <https://qcodes.github.io/Qcodes_contrib_drivers/>`__
+
+
+Broadbean
```
Comment (astafan8) [inline docs/community/related_projects.rst:31]: shall we also mention https://github.com/QCoDeS/lomentum ? or only when it becomes published on pypi etc etc ?

File: docs/community/related_projects.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+zhinst-qcodes can be installed with pip from PyPi
+with ``pip install zhinst-qcodes``
+The source code is hosted on Github
+`here <https://docs.zhinst.com/zhinst-qcodes/en/latest/>`__
+and the documentation can be found
+`here <https://github.com/zhinst/zhinst-qcodes/>`__
+
+
+
+
```
Comment (astafan8) [inline docs/community/related_projects.rst:77]: do we want to mention https://github.com/QuTech-Delft/qtt and https://github.com/QuTech-Delft/qilib ? or we'll let them mention themselves if they so wish?

**Review by astafan8 — APPROVED** [review]

File: docs/community/related_projects.rst (hunk 1 above)
Comment (jenshnielsen) [reply docs/community/related_projects.rst:31]: Even the repo is private at the moment so I would say no

File: docs/community/related_projects.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+zhinst-qcodes can be installed with pip from PyPi
+with ``pip install zhinst-qcodes``
+The source code is hosted on Github
+`here <https://docs.zhinst.com/zhinst-qcodes/en/latest/>`__
+and the documentation can be found
+`here <https://github.com/zhinst/zhinst-qcodes/>`__
+
+
+
```
Comment (astafan8) [inline docs/community/related_projects.rst:76]:
```suggestion
qtt (Qunatum Technology Toolbox)
--------------------------------

Quantum Technology Toolbox (QTT) is a Python-based framework 
for the tuning and calibration of quantum dots and spin qubits. 
QTT is the framework on which you can base your measurement 
and analysis scripts. QTT is based on Qcodes and the SciPy ecosystem.

`GitHub page with the repository <https://github.com/QuTech-Delft/qtt>`__

`Docs: <https://qtt.readthedocs.io>`__
```

File: docs/community/related_projects.rst (hunk 2 above)
Comment (astafan8) [reply docs/community/related_projects.rst:77]: just added a comment above to add just qtt

**Pull request merged**